dist/index.js

```diff
@@ -3005,14 +3006,21 @@ async function run() {
       }
     }
 
+    const extra_opts = [];
+    for (const token of extra_args.split(' ')) {
+      if (token) {
+        extra_opts.push(token)
+      }
+    }
+
```

Comment:
Is this necessary? Since we're not doing any additional validation here, why not just pass the intact string?

Reply:
Never mind, I see our usage of exec